python/py-emcee/Portfile

```diff
@@ -38,6 +37,9 @@ if {${name} ne ${subport}} {
 
     depends_test-append \
                     port:py${python.version}-nose
```

Comment:
I don't think the test phase can work, did you run the test-suite locally? When I updated this Portfile locally in my branch it seemed to me that upstream has switched to `pytest` and there are additional dependencies. 

I think it should be something like:
```
    depends_test-append \
                    port:py${python.version}-h5py \
                    port:py${python.version}-pytest \
                    port:py${python.version}-scipy

    test.run        yes
    test.cmd        py.test-${python.branch}
    test.target
    test.env        PYTHONPATH=${worksrcpath}/build/lib
```

but please double-check.